copilot-chat-prompts.el

```diff
@@ -27,156 +27,90 @@
-  the footer section from a multi-paragraph body). An exception is made for `BREAKING CHANGE`, which MAY also be used as a token.
-1. A footer's value MAY contain spaces and newlines, and parsing MUST terminate when the next valid footer
-  token/separator pair is observed.
-1. Breaking changes MUST be indicated in the type/scope prefix of a commit, or as an entry in the
-  footer.
-1. If included as a footer, a breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon, space, and description, e.g.,
-_BREAKING CHANGE: environment variables now take precedence over config files_.
-1. If included in the type/scope prefix, breaking changes MUST be indicated by a
-  `!` immediately before the `:`. If `!` is used, `BREAKING CHANGE:` MAY be omitted from the footer section,
-  and the commit description SHALL be used to describe the breaking change.
-1. Types other than `feat` and `fix` MAY be used in your commit messages, e.g., _docs: update ref docs._
-1. The units of information that make up Conventional Commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.
-1. BREAKING-CHANGE MUST be synonymous with BREAKING CHANGE, when used as a token in a footer.
-
----
-
-The following describes type.
-
-# type
-
-Must be one of the following:
-
-* **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-* **ci**: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs, GitHub Actions)
-* **docs**: Documentation only changes
-* **feat**: A new feature
-* **fix**: A bug fix
-* **perf**: A code change that improves performance
-* **refactor**: A code change that neither fixes a bug nor adds a feature
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-* **test**: Adding missing tests or correcting existing tests
-
----
-
-Here is the result of `git diff --cached`:
+Conventional Commits Specification (v1.0.0-beta.3)
+==================================================
+
+Overview:
+The Conventional Commits specification provides a standard for writing meaningful commit messages. It supports both human readability and machine automation. This convention is aligned with Semantic Versioning (SemVer).
+
+Commit Message Format:
+  <type>[optional scope]: <description>
+
+  [optional body]
+
+  [optional footer]
+
+Types:
+1. fix: for bug fixes (maps to PATCH in SemVer)
+2. feat: for new features (maps to MINOR in SemVer)
+3. BREAKING CHANGE: for breaking API changes (maps to MAJOR in SemVer). Must appear at the start of body or footer.
+
+Other allowed types (e.g. from Angular conventions):
+- chore:
+- docs:
+- style:
+- refactor:
+- perf:
+- test:
+- improvement: (recommended for non-feature/non-bugfix enhancements)
```

Comment:
The `improvement` type is likely not included. At least my commitlint doesn't allow it to pass. Content-wise, I think another type would work better. If it's to be included, I'd like to see some valid justification.

Reply:
@ncaq yep, the whole spec here is directly pulled from the official [Conventional Commit Spec](https://www.conventionalcommits.org/en/v1.0.0-beta.3/#specification). 

I haven’t personally used `improvement` before either, but I figured it’s in the spec, so leaving it in shouldn’t hurt. 

![image](https://github.com/user-attachments/assets/28a1e5eb-7a08-4e51-830a-54b9cd8789ac)

Reply:
That type is listed only in the beta version and appears to have been removed in the official release.
[Add improvement to allowed list. by OmgImAlexis · Pull Request #832 · conventional-changelog/commitlint](https://github.com/conventional-changelog/commitlint/pull/832)

I was also mistaken until now, but it seems that "beta" doesn't refer to a newer version - rather, the official release is actually more up-to-date in addition to being more stable.

I wonder why they don't remove the beta files from the repository?

I think we need to revise based on the officially released documentation.